lib/get-last-release.js

```diff
@@ -0,0 +1,40 @@
+const {promisify} = require('util');
+const {resolve} = require('url');
+const npmConf = require('npm-conf');
+const RegClient = require('npm-registry-client');
+const getClientConfig = require('./get-client-config');
+const getRegistry = require('./get-registry');
+
+module.exports = async ({publishConfig, name}, logger) => {
+  const config = npmConf();
+  const tag = (publishConfig || {}).tag || config.get('tag');
+  const {NPM_TOKEN, NPM_USERNAME, NPM_PASSWORD, NPM_EMAIL} = process.env;
+  const client = new RegClient(getClientConfig(config));
+  const registry = await getRegistry(publishConfig, name);
+
+  try {
+    const data = await promisify(client.get.bind(client))(resolve(registry, name.replace('/', '%2F')), {
```

Comment:
this is quite a lot at once to read easily, can you maybe split it up into multiple lines? At least move the value that `resolve(...)` returns into an own variable for better readability

Reply:
Done